packages/v4/src/content/design-guidelines/components/date-picker/date-picker.md

```diff
@@ -0,0 +1,143 @@
+
+## Elements
+
+<img src="./img/date-picker.png" alt="Date picker elements" width="382"/>
+
+1. **Date fields**: Displays a user’s selected date. Always fill the date field with a placeholder date format (MM/DD/YYYY) or a pre-selected date before users make their selection via calendar or keyboard. The date field should never be empty.
+2. **Calendar icon**: Opens a calendar upon selection.
+3. **Selection arrows**: Allow users to navigate between months.
+4. **Calendar**: Allows users to select and navigate between days, months, years. Restricts selections by disabling certain dates.
+5. **Selected date**
+6. **Current date (today’s date)**: Indicates the day users access the calendar. Always highlighted.
+7. **Hovered state**
+8. **Disabled state**
+
+## Usage
+Use date pickers to enable users to select or input a date for use cases like item filtration or scheduling in a form.
+
+### Single date selection
+A date picker allows users to select a single date. When a user enters a new date by date field input or calendar selection, the new date automatically replaces any placeholder or previously-selected value in the date input field.
+
+TO DO (link) For more guidance on writing date formats, see the [content](#date-format) section.
+
+### Date range selection
+Two date pickers can be used together to specify a date range. The first one functions as a start (“from”) date and the second one functions as an end (“to”) date. By default, when a user selects a start date, the end date will automatically display as the next calendar day.
+
+<img src="./img/Date range selection.png" alt="Date range selection - elements" width="482"/>
+
+**Selected start date**
+**Time range**
+**Selected end date**
+
+### Date and time range selection
+Combine a date picker and time picker when you want users to select a range of times across multiple days.
+
+**Behavior**
+When date pickers and time pickers are combined, each control’s behavior will remain the same. After users select the start date and time, the end date and time will be automatically set to the same hour of the next day.
+
+Depending on your use case, time pickers and date pickers can be arranged in two main ways: Horizontal alignment and vertical alignment (in this case labels are recommended).
+
+**Horizontal**
+
+<img src="./img/Time and Date picker horizontal.png" alt="Example of a horizontal date and time range selection" width="550"/>
+
+For horizontally arranged date and time pickers, connect each combined date and time picker with "to" so that users know which date defines which point of the date range. Always arrange each date and time picker in chronological order from left to right, with the start picker on the left and the end picker on the right.
+
+**Vertical with labels**
+
+<img src="./img/Time and Date picker vertical with labels.png" alt="Example of a vertical date and time range selection with labels" width="242"/>
```

Comment:
Can you change the labels here so that they read "Start" and "End" without the "s" at the end of each of them?